### PR TITLE
Uppercase the first letter in the printing message

### PR DESCRIPTION
**Pull request by bocharsky-bw**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: symfony/framework-bundle/3.3/Makefile [hunk 1]
```diff
@@ -16,7 +16,7 @@ cache-warmup: cache-clear
 ifdef CONSOLE
 	@$(CONSOLE) cache:warmup
 else
-	@printf "cannot warmup the cache (needs symfony/console)\n"
+	@printf "Cannot warmup the cache (needs symfony/console)\n"
```
Comment (javiereguiluz) [inline symfony/framework-bundle/3.3/Makefile]: If I'm right, "warmup" word doesn't exist. It's `warm-up` as a noun and `warm up` as a verb, so in this case it should be "Cannot warm up the cache ..."

Comment (bocharsky-bw) [reply symfony/framework-bundle/3.3/Makefile]: 👍  for "warm up"

Comment (fabpot) [reply symfony/framework-bundle/3.3/Makefile]: @bocharsky-bw Can you make the change?

Comment (bocharsky-bw) [reply symfony/framework-bundle/3.3/Makefile]: Done

**Pull request merged**